### PR TITLE
{chem}[intel/2020a] libctl v4.0.0

### PR DESCRIPTION
**Pull request by ekieffer**
(created using `eb --new-pr`)

requires [~#11382~ ] (Guile)

### CODE REVIEW

**Comment by ekieffer** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-001 - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/ec3fa9d1aa3682baca85c4bdb4cbb25b for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 3.8.2
See https://gist.github.com/97c5ced9a1c6190be14431babc7f4f1e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node378.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c1c1d455320ccb29ba2e0f1253412109 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node100.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/9079ba6d54634dbaa201328a16fb5669 for a full test report.

**Review by lexming — DISMISSED** [review]
@ekieffer Thanks for the PR, the build went fine. I would make a couple minor changes before merging it, please consider the following suggestions

**Comment by verdurin** [comment]
@ekieffer any thoughts on the comments from @lexming ?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/l/libctl/libctl-4.0.0-intel-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'libctl'
+version = '4.0.0'
+
+homepage = 'https://github.com/stevengj/libctl'
+description = """libctl is a free Guile-based library implementing flexible control files for scientific simulations."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'optarch': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/l/libctl/libctl-4.0.0-intel-2020a.eb:10]: No need to enable `optarch`, that's the default

```suggestion
toolchainopts = {'pic': True}
```

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2625.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/462132dd1f01ac4f3c6733861ed74710 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3516.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/b6a29486e93a8228b4364528fe76007e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ekieffer!

**Pull request merged**